ui/pages/confirmations/components/confirm/network-change-toast/network-change-toast-inner.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+
+import { Box } from '../../../../../components/component-library';
+import { Toast } from '../../../../../components/multichain';
+import {
+  getLastInteractedConfirmationInfo,
+  setLastInteractedConfirmationInfo,
+} from '../../../../../store/actions';
+import { getCurrentChainId } from '../../../../../selectors';
+import { NETWORK_TO_NAME_MAP } from '../../../../../../shared/constants/network';
+import { useI18nContext } from '../../../../../hooks/useI18nContext';
+
+const MILLISECONDS_IN_ONE_MINUTE = 60000;
+const MILLISECONDS_IN_FIVE_SECONDS = 5000;
```

Comment:
Rather than naming these for their value, should we instead name them for their usage?

Such as `CHAIN_CHANGE_THRESHOLD_MILLISECONDS` and `TOAST_TIMEOUT_MILLISECONDS`?

Reply:
Name for value also look good to me.

Reply:
I'm thinking of maintainability and readability.

So if we have a requirement to change the timeout or threshold in future, there's no indication which value is which without checking the code paths first to see where each is referenced.

In addition, when we do change the value, we have to rename it every time.

We could retain all the same information with:

```js
const CHAIN_CHANGE_THRESHOLD_MILLISECONDS = 60 * 1000; // 1 Minute
const TOAST_TIMEOUT_MILLISECONDS = 5 * 1000; // 5 Seconds
```